components/kyma-environment-broker/README.md

```diff
@@ -32,3 +32,49 @@ This table lists the environment variables, their descriptions, and default valu
 | **APP_DATABASE_PORT** | Defines the database port. | `5432` |
 | **APP_DATABASE_NAME** | Defines the database name. | `broker` |
 | **APP_DATABASE_SSL** | Specifies the SSL Mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
+
+
+## Development
+
+This section presents how to add or remove functionality for disabling components.
```

Comment:
I would use "runtime component". Somebody could think that a component is a part of KEB.